controllers/TodoControl.js

```diff
@@ -0,0 +1,75 @@
+const { Todo } = require('../models')
+
+class TodoControl {
```

Comment:
Biasanya nama classnya

```suggestion
class TodoController {
```